modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -7612,6 +7613,128 @@ public void testPostObjectEntryWithKeywordsAndTaxonomyCategoryIdsWhenCategorizat
 		}
 	}
 
+	@Test
+	public void testPostPutCustomObjectEntryWithPermissions() throws Exception {
```

Comment:
Did you find a pattern that tests Post and Put using the same test method and that uses this prefix `testPostPut`?
I prefer to separate them.

Reply:
@marcelabc I saw other methods in this same test class following this pattern:

Example: [testPatchPutCustomObjectEntryExternalReferenceCode](https://github.com/liferay-objects/liferay-portal/blob/db30ccde22468d67701614129f79463b9dd2f430/modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java#L6879)

Reply:
Thank you!